pkg/suggestion/v1alpha3/chocolate/base_chocolate_service.py

```diff
@@ -1,11 +1,11 @@
-import json
-import numpy as np
 import chocolate as choco
 import logging
 import base64
 
-from pkg.suggestion.v1alpha3.internal.search_space import *
-from pkg.suggestion.v1alpha3.internal.trial import *
+from pkg.suggestion.v1alpha3.internal.search_space import (
+    INTEGER, DOUBLE, CATEGORICAL, DISCRETE, MAX_GOAL,
```

Comment:
Can we do this in a better way to look more clean? we might have more constants. May be wrap this?

Reply:
You mean that you want to import all constants, right?
Another solution is to define `__all__` attribute like [this revision](https://github.com/kubeflow/katib/commit/f99736db4c79009e8988b498090116b5b09230bb). But as the discussion of https://github.com/kubeflow/katib/pull/1133#discussion_r406294910, it is known as an anti-pattern in Python.

Reply:
@c-bata What do you think about creating file: `constant.py` under `/internal/` folder ?
And insert all constant there? I think it will be easier to have all constant in one file

Reply:
It sounds good 👍  @johnugeorge What do you think?

Reply:
Yes. SGTM